international-telephone-input/assets/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>International telephone input</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/17.0.8/css/intlTelInput.css"
+    />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/17.0.8/js/intlTelInput.min.js"></script>
+  </head>
+  <body>
+    <div class="container">
+      <h1>International telephone input</h1>
+      <p>
+        This example shows how to process and parse international phone numbers with the
+        <a href="https://github.com/jackocnr/intl-tel-input"><code>intl-tel-input</code></a>
+        library and detect invalid phone numbers with the
+        <a href="https://twilio.com/docs/lookup/api">Twilio Lookup API</a>.
+      </p>
+      <div>
+        <form id="login" onsubmit="process(event)">
```

Comment:
I'm personally allergic to `on*` handlers in the HTML. Could we select the form below and use `addEventListener` instead? To that note, the id of "login" seems weird here. Could we have `<form id="lookup">` for example, and then in the JS:

```js
const form = document.getElementById('lookup');
form.addEventListener('submit', process);
```

to add the event listener.

Reply:
absolutely, I'm still learning HTML/JS best practices - can you explain why you're allergic to `on*` handlers like I did it? Is it preference or are there reasons for using the `addEventListener`?

Reply:
Yeah, this might seem odd in a world of React where event listeners are added on the JSX. Using `addEventListener` over inline event handlers is known as unobtrusive JavaScript, and is designed to keep a separation between presentation (the HTML) and behaviour (the JavaScript). It also has benefits, like the ability to add more than one listener though that is not required in this case.

I didn't realise, but [unobtrusive JavaScript even has a wikipedia page](https://en.wikipedia.org/wiki/Unobtrusive_JavaScript)!

Reply:
awesome, ty for the explanation!